### ISSUE TITLE
[Feature Request]: Show chapter title on seek bar

### ISSUE THREAD

**Issue by R1Monkey** [FreeTubeApp/FreeTube#2829]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

The chapters are not visible on the timeline 
they appear in a window below the video but not on the main timeline 
![Screenshot 2022-11-04 at 11 45 13 PM](https://user-images.githubusercontent.com/117497591/200099434-a791044e-a5b1-4209-aa96-dbfe785a3c60.jpg)
![Screenshot 2022-11-04 at 11 45 21 PM](https://user-images.githubusercontent.com/117497591/200099435-1411edb8-f5f9-4433-984d-80e0d2a6860f.jpg)

### Proposed Solution

It should show the chapter title when hovering over timeline

### Issue Labels

ease of use improvement, improvement to existing feature

**Comment by MarmadileManteater** [comment]
For full clarity, this isn't a feature that stopped working. Chapters are a new feature as of [`v0.18.0 Beta`](https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.18.0-beta) (see FreeTubeApp#2224 for the exact changes). It is worth noting that they are visually different than they appear in YT currently, but AFAIK, the chapter title hasn't ever been displayed when scrubbing the progress bar in FreeTube.

**Comment by ChunkyProgrammer** [comment]
It might be a good idea to also display sponsorblock category when hovering over one.

Ex:
Video thumbnail 
Chapter Title (bold)
Sponsorblock Category 
Timestamp

**Comment by BarbzYHOOL** [comment]
It is an extremely useful feature that is available on android tv box from the official youtube API

It shows the chapter but ALSO allows to quickly skip to that segment of the video. It even (since recently) displays a tiny thumbnail of the segment if I recall right.

**Comment by ExtRIELICi** [comment]
I completely agree with this. I have no idea why they decided to place it at the bottom of the video, it just adds more clicks and effort for something that should be easy and quick.

**Comment by kommunarr** [comment]
I don't know if there's that clean of a way to do this currently. Creating a custom `FtSeekBar` component that extends the Shaka seek bar one and tossing in our extra fields?  Injecting it with the [`content` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/content)? @absidue

**Comment by absidue** [comment]
@kommunarr I feel like I've explained this before but shaka-player uses the google closure compiler which is the only javascript bundler that actually handles classes properly, which means that the class you see in the source code will not match the one in the bundle that FreeTube uses, you will only have access to the classes, properties and methods are explicitly exported (@protected, @export), anything tagged with private will definitely not be available from the FreeTube side.

You'll have to create a new seek bar from scratch or copy over the one from upstream as well as all of it's utilities that aren't exposed in the bundle.

**Comment by QuakeString** [comment]
Recently I'm watching a lot of coding tutorials and it's really a need feature. Every time I have to press "S" then scroll down to find the chapter name.

**Comment by SupremeMuhit** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc 
Bro keeps calling everyone a duplicator, but he’s not adding any actual features. A lot of people really want this feature to be added😑
